### DISCUSSION TITLE
Dominio

### DISCUSSION THREAD

**Discussion by andreparelho** [rafaelleitedasilva/blogtext#5] [Polls]
Criada discussão para definir o dominio do projeto.

**Comment by EdilsonBaggio** [comment]
Acho que combina mais com o projeto.

**Reply by rafaelleitedasilva** [reply]
Pior que eu fiquei em dúvida entre esses dois, mas PoemBook dá uma ideia de ser um site para livros de poema. Ou só eu tenho essa impressão? kkkkkkkk
